src/MapPanel.jsx

```diff
@@ -14,6 +15,10 @@ export class MapPanel extends React.Component {
     return (
       <section className="map-panel">
         <ActionBar/>
+        { this.props.error ?
+          <ErrorBlock text="There was a problem executing your search" /> :
```

Comment:
this works, but you can also do this to avoid that ternary

```
{ this.props.error &&
          <ErrorBlock text="There was a problem executing your search" />
}
```